operations/CSM_product_management/Run_NCN_Personalization.md

```diff
@@ -0,0 +1,167 @@
+* Path to the Ansible play to run in the repository,
+* Commit ID in the repository for CFS to pull and run on the nodes.
+
+Products may supply multiple plays to run, in which case multiple configuration
+layers must be created.
+
+<a name="ncn_personalization_determine_existence"></a>
+## Determine if NCN Personalization CFS Configuration Exists
+
+If upgrading a product to a new version, an NCN personalization configuration in
+CFS should already exist. By default the configuration is named `ncn-personalization`.
+
+1. Determine if a configuration already exists.
+   ```bash
+   ncn# cray cfs configurations describe ncn-personalization --format json > ncn-personalization.json
+   ```
+
+If the configuration exists, the `ncn-personalization.json` file will be
+populated with configuration layers. If it does not exist, the file will be
+empty and the command will respond with an error.
+
+<a name="ncn_personalization_add_layer"></a>
+## Add Layer(s) to the CFS Configuration
+
+1. Add a configuration layer to the `ncn-personalization.json` file.
+   1. If the `ncn-personalization.json` file is empty, overwrite the file with
+      the configuration layer(s) information gathered from the product that is
+      configuring the NCNs. Use the [sample file with a single layer](../configuration_management/Configuration_Layers.md#configuration_layer_example_configuration_single)
+      as a template.
+   1. If a CFS configuration exists with one or more layers, add (or replace)
+      the corresponding layer entry(ies) with the configuration layer
+      information gathered for this specific product. For example:
+   ```bash
+   ncn# cat ncn-personalization.json
+   {
+      "layers": [
+         # ...
+         {
+            "name": "<product-release-etc>",
+            "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/<product>-config-management.git",
+            "playbook": "site.yml",
+            "commit": "<git commit>"
+         },
+         # ... 
+      ]
+   }
+   ```
+   CFS executes the configuration layers in order. Consult the product
+   documentation to determine if its configuration layer requires special
+   placement in the layer list.
+
+<a name="ncn_personalization_update_cfs_configuration"></a>
+### Create/Update the NCN Personalization CFS Configuration Layer
+
+1. Upload the configuration file to CFS to update or create the
+   `ncn-personalization` CFS configuration.
+   ```bash
+   ncn# cray cfs configurations update ncn-personalization --file ncn-personalization.json --format json
+   {
+      "lastUpdated": "2021-07-28T03:26:01Z",
+      "layers": [
+         { ... layer information here ... },
+      ],
+      "name": "ncn-personalization"
+   }
+   ```
+   > **WARNING**:
+   > When running the above step, the CFS session may fail due to the node's
+   > sshd configuration being corrupted by a different process. Run the
```

Comment:
What would the failure look like?  Might want to note that here to make it more obvious when to do this.